cybershell/fingerprint.py

```
@@ -0,0 +1,649 @@
+        
+        return fingerprint
+    
+    def _passive_fingerprint(self, fingerprint: TargetFingerprint, target: str):
+        """Passive fingerprinting using single request"""
+        try:
+            # Make initial request
+            response = requests.get(
+                target,
+                timeout=self.timeout,
+                headers={'User-Agent': 'Mozilla/5.0 (compatible; SecurityScanner/1.0)'},
+                verify=False,
```

Ruff (0.12.2)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)